pkg/kubernetes/schema_test.go

```diff
@@ -43,12 +43,14 @@ func TestMixin_ValidatePayload(t *testing.T) {
 		error string
 	}{
 		{"install", "testdata/install-input.yaml", true, ""},
+		{"install-with-kubeconfig", "testdata/install-input-with-kubeconfig.yaml", true, ""},
 		{"upgrade", "testdata/upgrade-input.yaml", true, ""},
 		{"invoke", "testdata/invoke-input.yaml", true, ""},
 		{"uninstall", "testdata/uninstall-input.yaml", true, ""},
 		{"install-bad-wait-flag", "testdata/install-input-bad-wait-flag.yaml", false, "install.0.kubernetes.wait: Invalid type. Expected: boolean, given: string"},
 		{"install-no-manifests", "testdata/install-input-no-manifests.yaml", false, "install.0.kubernetes: manifests is required"},
 		{"install-bad-outputs", "testdata/install-input-bad-outputs.yaml", false, "install.0.kubernetes.outputs.0: resourceType is required\n\t* install.0.kubernetes.outputs.0: resourceName is required\n\t* install.0.kubernetes.outputs.0: jsonPath is required"},
+		{"install-with-kubeconfig", "testdata/install-input-with-kubeconfig.yaml", true, "install.0.kubernetes.outputs.0: resourceType is required\n\t* install.0.kubernetes.outputs.0: resourceName is required\n\t* install.0.kubernetes.outputs.0: jsonPath is required"},
```

Comment:
I think this test cases was added accidentally? It doesn't appear to be valid.

Reply:
The execute command looks to include the install command, the first line of `buildExecuteCommand` calls `buildInstallCommand` so the `kubeConfig` field flows down does it not?

Reply:
When I said invalid I was referring to the error that it is expecting to be returned. Shouldn't this test case pass? But instead it has the same expected error message as the "install-bad-outputs" test case above it.